### ISSUE TITLE
Не могу выйти в Интернет: Ошибка 451

### ISSUE THREAD

**Issue by Teashrock** [podivilov/runet-blackout#34]
**Нафиг вашу изоляцию Рунета, верните свободный Интернет!**

**ДАЁШЬ СВОБОДУ СЛОВА И МЫСЛИ! ДАЁШЬ СВОБОДУ ВОЛИ! ДАЁШЬ СПРАВЕДЛИВОСТЬ!**

**Comment by podivilov** [comment]
Это не относится к сетям (1-4 уровня), но ошибка HTTP интересная.

**Comment by f14a2cd** [comment]
Офигеть. 
451 ошибка обычно применяется в LiveJournal, когда РКН добирается до журнала юзера
